src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/IndexManagementPlugin.kt

```diff
@@ -59,32 +62,27 @@ import org.elasticsearch.threadpool.ThreadPool
 import org.elasticsearch.watcher.ResourceWatcherService
 import java.util.function.Supplier
 
-internal class IndexStateManagementPlugin : JobSchedulerExtension, ActionPlugin, Plugin() {
+internal class IndexManagementPlugin : JobSchedulerExtension, ActionPlugin, Plugin() {
 
     private val logger = LogManager.getLogger(javaClass)
-    lateinit var indexStateManagementIndices: IndexStateManagementIndices
+    lateinit var indexManagementIndices: IndexManagementIndices
     lateinit var clusterService: ClusterService
 
     companion object {
-        const val PLUGIN_NAME = "opendistro-ism"
-        const val ISM_BASE_URI = "/_opendistro/_ism"
+        const val PLUGIN_NAME = "opendistro-im"
+        const val OPEN_DISTRO_BASE_URI = "/_opendistro"
+        const val ISM_BASE_URI = "$OPEN_DISTRO_BASE_URI/_ism"
         const val POLICY_BASE_URI = "$ISM_BASE_URI/policies"
-        const val INDEX_STATE_MANAGEMENT_INDEX = ".opendistro-ism-config"
-        const val INDEX_STATE_MANAGEMENT_JOB_TYPE = "opendistro-managed-index"
+        const val INDEX_MANAGEMENT_INDEX = ".opendistro-ism-config"
```

Comment:
Not sure why change this to INDEX_MANAGEMENT_INDEX, it should still for state management right

Reply:
We are adding more types of index management jobs into this config index; Rollups and Transforms. So this is not just for ISM anymore. Unfortunately we cannot change the index name itself in `.opendistro-**ism**-config` so this will just have to be a legacy artifact, but we can rename the variable at least.